src/meta/src/rpc/service/ddl_service.rs

```diff
@@ -467,6 +467,7 @@ where
         for fragment in fragment_graph.fragments.values_mut() {
             mview_count += fill_mview_id(fragment.node.as_mut().unwrap(), mview_id);
         }
+        let fragment_graph = fragment_graph;
```

Comment:
```suggestion
```

Reply:
This's for making it immutable, so we won't fear that the variable will unexpectedly change after this line. 🥺